### PR TITLE
Add eslint config:

### PR DESCRIPTION
**Pull request by denibulkashvili**
* npm run lint to run linter

### CODE REVIEW

File: .eslintrc.json [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+        "react/jsx-closing-bracket-location": 1,
+        "react/jsx-curly-spacing": [2, "always"],
+        "react/jsx-indent-props": [1, 2],
+        "react/jsx-no-undef": 1,
+        "react/jsx-uses-react": 1,
+        "react/jsx-uses-vars": 1,
+        "react/wrap-multilines": 1,
+        "react/react-in-jsx-scope": 1,
+        "react/prefer-es6-class": 1,
+        "react/jsx-no-bind": 1
```
Comment (sharadJay) [inline .eslintrc.json]: do we need to specify so many configs? I thought Airbnb would do that for us.

Comment (denibulkashvili) [reply .eslintrc.json]: We don't but I've been meaning to ask you whether we want to tweak or change some of them

File: package.json [hunk 2]
```diff
@@ -9,15 +9,25 @@
     "dev": "npm run heroku-postbuild && npm run start",
     "heroku-postbuild": "cd client && npm run build",
     "test": "jest",
-    "posttest": "cd client && npm run build && npm test"
+    "posttest": "cd client && npm run build && npm test",
+    "lint": "./node_modules/.bin/eslint src",
```
Comment (sharadJay) [inline package.json]: This should work even without ./node_modules/.bin/ I think. Can you try? http://2ality.com/2016/01/locally-installed-npm-executables.html

File: .eslintrc.json (hunk 1 above)
Comment (sharadJay) [reply .eslintrc.json]: nope. I would rather go with standard airbnb and then tweak I want to do it differently

**Pull request merged**